src/web_ui/__init__.py

```diff
@@ -49,7 +60,8 @@ def index():
     try:
         net_env = NetworkEnv()
         # An ugly-ish way to combine two dicts, in order to get the interface names on the same level
-        network_interfaces = [{**v, **{'name': k}} for k, v in net_env.interfaces.items()]
+        network_interfaces = [{**v, **{'name': k}}
+                              for k, v in net_env.interfaces.items()]
```

Comment:
Over-zealous linting needs to be less zealous and allow for line length up to 120 chars :)